inductiva/coastal/README.md

```diff
@@ -0,0 +1,93 @@
+# Coastal dynamics
+
+Inductiva API supports simuulation of coastal dynamics via two simulators:
+[SWASH](https://swash.sourceforge.io/) and [XBeach](https://oss.deltares.nl/web/xbeach/).
+These simulators solve the [shallow-water equations](https://en.wikipedia.org/wiki/Shallow_water_equations),
+a reduced version of the Navier-Stokes equations valid for small depth-to-length
+ratios (i.e., shallow water).
+
+## Coastal area scenario
```

Comment:
I think that, ideally, we could introduce the coastalarea scenario with just one simple bathymetry, and then have another section just on bathymetries because they are a really really amazing class.